src/links/links.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { Link, Prisma } from '@prisma/client';
+import { PrismaService } from 'src/prisma/prisma.service';
+
+@Injectable()
+export class LinksService {
+  constructor(private prisma: PrismaService) {}
+
+  async findOne(id: Link['id']): Promise<Link | null> {
+    return this.prisma.link.findFirst({
+      where: { id },
+    });
+  }
+
+  async findAll(): Promise<Link[] | null> {
```

Comment:
**suggestion**: Remove the redundant `await` in `return await this.prisma.link.findMany();`.